src/components/__tests__/page-details.test.jsx

```diff
@@ -1,16 +1,23 @@
 /* eslint-env jest */
-import timers from 'node:timers/promises';
+import { render, waitFor } from '@testing-library/react';
 import PageDetails from '../page-details/page-details';
-import { shallow } from 'enzyme';
 import simplePage from '../../__mocks__/simple-page.json';
+import { TestApiContextProvider } from '../../__mocks__/api-context-provider';
 import WebMonitoringDb from '../../services/web-monitoring-db';
 
+jest.mock('../change-view/change-view');
+
 describe('page-details', () => {
   // Change string values to date objects so they're parsed correctly
   simplePage.versions.forEach(version => {
     version.capture_time = new Date(version.capture_time);
   });
-  const match = { params: { pageId: simplePage.uuid } };
+  const match = {
+    params: {
+      pageId: simplePage.uuid,
+      change: `${simplePage.versions[1].uuid}..${simplePage.versions[0].uuid}`,
+    }
+  };
```

Comment:
With full rendering, `<PageDetails>` tries to redirect and update the URL when change info is missing. This wasn’t an issue with Enzyme’s shallow rendering, where it would never run the lifecycle events that led to the redirect.

We *should* add some real tests of those redirects, but for now it’s much more straightforward to set up the tests so they don’t cause redirects (especially since the shallow rendering wouldn’t have been testing that anyway).